webrtc.html

```diff
@@ -16278,11 +16278,6 @@ <h3>
   try {
     if (description) {
       await pc.setRemoteDescription(description);
-      // if we got an offer, we need to reply with an answer
-      if (description.type == 'offer') {
-        await pc.setLocalDescription();
-        signaling.send({description: pc.localDescription});
-      }
```

Comment:
The conferencing server may send an offer to receive simulcast. So applications may need to handle this.

Also, to handle renegotiation for stuff I don't think we should customize the negotiation handling code.

I vote against merging this.

Reply:
We could add something to the comment:
```
// if we got an offer, we need to reply with an answer. This might
// happen e.g. if a server later wants to send media back to the client.
```